### PR TITLE
Created first post for Rachel Anne

### PR DESCRIPTION
**Pull request by rachelanniexo**
Hi! This is just a first post I created with a little info about myself.

### CODE REVIEW

File: _posts/rachelanniexo/2018-05-17-rachelanniexo-first-post.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: rachelanniexo
+title: "Rachel Anne's first post"
+---
+Hi! My name is Rachel Anne and I'm a rising 2nd-year MSIS student. I'm primarily focused on UX, so I am (loosely) following the the HCI track. I work in the SILS library and am an officer for ILSSA, ASIS&T @ UNC, and FLUX (Future Leaders in User eXperience - the new UX student org started by Cami Goray).
```
Comment (eah13) [inline _posts/rachelanniexo/2018-05-17-rachelanniexo-first-post.md]: Hello and welcome! 👾

Comment (rachelanniexo) [inline _posts/rachelanniexo/2018-05-17-rachelanniexo-first-post.md]: Fun fact I didn't include: I'm really good at finding four-leaved clovers :shamrock:

**Comment by rachelanniexo** [comment]
Sorry for being the last to submit.

**Comment by rachelanniexo** [comment]
Additionally, here is my Basics of Turtle assignment. I did figure out how to do the overlapping triangles and added a large triangle underneath to make a gem shape. I was trying to recreate a logo, but I couldn't figure out the angles (or maybe slopes??) of the lower gem triangles, so I just made the three of them into one big one. https://trinket.io/python/f02af10bd6

**Pull request merged**